packages/strapi-helper-plugin/lib/src/app.js

```diff
@@ -8,11 +8,25 @@
 import React from 'react';
 import { Provider } from 'react-redux';
 
-import App, { bootstrap } from 'containers/App'; // eslint-disable-line
+import App from 'containers/App'; // eslint-disable-line
 
 import configureStore from './store';
 import { translationMessages } from './i18n';
 
+const tryRequire = (bootstrap = false) => {
```

Comment:
The argument should be the name of the file: `fileName`. So you can call the function in a more generic way: `tryRequire('bootstrap');` and `tryRequire('pluginRequirements');`.

Reply:
Tried and it didn't work that's why there were a `const path = 'bootstrap';` that I forgot to delete